### PR TITLE
#927-improve-layout-of-form-fields

### LINKED ISSUE #927: Improve Layout of Form Fields - Activate WP Form
### Description

We want to update the layout of all the forms to be cleaner, see specific details below.

Epic: #899 

### Acceptance Criteria

- this modal looks weird when it's so skinny
- order things as project lead and project manager on one line
- then order things as start date and confirmation on another line
- ensure things are still responsive and dynamically resize with available screen real estate

### Proposed Solution

N/A

### Mocks

image

### PR DESCRIPTION
**Pull request by MWalve**
## Changes

Changed the layout of the form for the Work packages, such that it was no longer skinny, and the fields were beside each other

## Notes

Still think it looks a bit off, width and height wise, and the title Activate's font size still seems small compared to the layout

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #927

### CODE REVIEW

File: src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx [hunk 1]
```diff
@@ -66,82 +66,90 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
...
+                      <FormControlLabel value={1} control={<Radio />} label="Yes" />
+                      <FormControlLabel value={0} control={<Radio />} label="No" />
+                    </RadioGroup>
+                  </>
+                )}
+              />
+            </Grid>
+          </Grid>
         </DialogContent>
+
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx:152]: delete line pls

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Looks Good! Looks like u added an extra line but other than that looks good!

**Review by Peyton-McKee — APPROVED** [review]
looks good

**Review by RChandler234 — APPROVED** [review]
LGTM

**Pull request merged**